Cargo.toml

```diff
@@ -112,10 +112,8 @@ crossbeam-channel = "0.5.6"
 [target."cfg(windows)".dependencies]
 winapi = "0.3.9"
 
-# We optimize the release build for size.
```

Comment:
IMO this comment can stay.

Also it may be worth leaving a comment explaining why we don't do LTO anymore.